server/templates/obs.html

```diff
@@ -112,4 +112,18 @@ <h4>Products</h4>
         </div>
     {% endfor %}
     {% include 'pagination.html' %}
+
+    {% if owner %}
+    <div>
```

Comment:
The nodes with the same class mean things of the same type.
You should assign the "admin" class to the "div" node and next use cascading in your style sheet.

```css
.admin fieldset { ... }
.admin legend { ... }
```
or in SASS:
```sass
.admin:
    fieldset:
        ...
        legend:
            ...
```

Reply:
We don't use SASS in this project (yet). I _think_ I've updated the styles as you wanted, but please check if this is more or less how you wanted it to be. CSS is a very trial and error (and mostly error) territory for me.